src/Codeception/Module/REST.php

```diff
@@ -1587,8 +1602,8 @@ public function seeBinaryResponseEquals($hash, $algo = 'md5')
      * ```
      * Opposite to `seeBinaryResponseEquals`
      *
-     * @param $hash the hashed data response expected
-     * @param $algo the hash algorithm to use. Default md5.
+     * @param string $hash the hashed data response expected
+     * @param string $algo the hash algorithm to use. Default md5.
```

Comment:
Is it your intention to add typehints in this same PR?
If so, there are a few that are missing throughout the file.

Reply:
My goal wasn't to document all methods, the problem with this particular method was that it had @param annotation without type, so PHPStorm highlighted it as `missing class the` error.